pandas/core/tools/datetimes.py

```diff
@@ -400,6 +403,18 @@ def _convert_listlike_datetimes(
             arg = arg.tz_convert(None).tz_localize("utc")
         return arg
 
+    elif isinstance(arg_dtype, ArrowDtype) and np.issubdtype(
```

Comment:
Could you use `arg_dtype.kind == "M"` instead of `np.issubdtype`?

Reply:
Thx.